domenetjenester/iay/src/main/java/no/nav/foreldrepenger/abakus/domene/iay/kodeverk/RelatertYtelseTema.java

```diff
@@ -11,10 +11,6 @@
 
```

Comment:
Trengs denne klassen? har vi ikke allerede en ytelsetype klasse? Hvis det er lokalt for en enkelt tjeneste, bør det heller være en enum?

Reply:
Relatertytelsetema er fra den tiden kodeverksdrevet oppslag var hot, også i anti-corruption.
Dette er hovedtema-kodene vi får fra Arena og Infotrygd. De skal oversettes til RelatertYtelseType, men for alt man vet finnes det kodeliste_relasjon her 
Tror ikke de brukes i lagring eller at det fins noen entiteter som referer til dem, slik at de kan gjemmes i respektive termoversettere

Reply:
Tror den kan konverteres til enum i abakus. Brukes strengt ikke som kodeliste